nautobot_chatops/dispatchers/base.py

```diff
@@ -216,11 +216,22 @@ def ask_permission_to_send_image(self, filename, action_id):
 
     # Send various content to the user or channel
 
-    def send_markdown(self, message, ephemeral=False):
+    def send_markdown(
+        self,
+        message,
+        ephemeral=settings.PLUGINS_CONFIG["nautobot_chatops"]["send_all_messages_private"],
```

Comment:
Design-wise I'm not a big fan of having a function's argument defaults depend on a dynamic settings value, in particular as it makes unit testing a bit more difficult  (using Django's `override_settings` decorator probably won't work as expected here). I'd prefer a pattern more like:


```python
def send_markdown(self, message, ephemeral=None):
    if ephemeral is None:
        ephemeral=settings.PLUGINS_CONFIG["nautobot_chatops"]["send_all_messages_private"]
   ...
```

even though it means writing a bit more code.